### PR TITLE
trancate text

### LINKED ISSUE #9842: Content card layout overlaps
## Observed behavior
Narrower content cards in the grid (in in-between display sizes) show overlapping content.

## Errors and logs

**Card with overlap**
Screen Shot 2022-11-18 at 2 54 46 PM

**Slightly wider card, title only takes up 2 lines, not 3 - no overlap**
Screen Shot 2022-11-18 at 2 54 35 PM

## Expected behavior
Title overflow should be truncated to reduce layout conflicts, so that relevant content can be displayed without overlaps

## User-facing consequences
May be difficult to read information

## Steps to reproduce
1. Ensure you have the "Kolibri QA Channel" imported
2. Open up "Make 10, grids and number bonds"
3. Shrink the browser window enough that the cards become narrower, but not so much that the side-bar disappears. 
4. Observe the title runs into 3 lines and content begins overlapping 

## Context
0.15.x, Chrome

### PR DESCRIPTION
**Pull request by AllanOXDi**
## Summary
This fixes content card layout overlaps to reduce layout conflicts.

## References

https://user-images.githubusercontent.com/103313919/204340903-5404172d-4c8f-4da5-be84-8b4127be210b.mp4

closes #9842

## Reviewer guidance
- Ensure you have the "Kolibri QA Channel" imported
- Open up "Make 10, grids and number bonds"
- Shrink the browser window enough that the cards become narrower and observe.

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: package.json
Comment (marcellamaki) [inline package.json]: Although this volta version is necessary for your local dev environment to work properly, you don't want to commit it when you make changes.

File: kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue
Comment (marcellamaki) [inline kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue:48]: Is it necessary for this just to be "1"? When we had looked at it together, it seemed like having the max lines set to 2 would be acceptable and not cause a conflict. I'm worried that having only one line might result in not enough text being displayed for some longer titles. If you have a screenshot of an example that demonstrates why "1" is better, that would be helpful to include in the PR!

**Review by marcellamaki — CHANGES_REQUESTED** [review]
Hi @AllanOXDi - thanks for working on this! I added a question and a small fix request to remove the volta reference from your `package.json`.

File: kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue
Comment (AllanOXDi) [reply kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue:48]: Hi Marcella! I have tried setting it to 2. But I realised that setting it to one is actually better. 
[screen-capture (3).webm](https://user-images.githubusercontent.com/103313919/204302857-132c4ef4-5f3e-41a5-b37b-b58cc566ba7d.webm)

Please don't merge it yet today as I explore more alternatives.

**Review by marcellamaki — APPROVED** [review]

**Comment by marcellamaki** [comment]
Thanks for updating the `package.json` @AllanOXDi - I'll go ahead and merge this, and if we come to more ideas about spacing in the future, or with design input, we can update it.

**Pull request merged**
